OasisDEX/oasis-borrow
# Use library for multiply calculations

# [Use common library for calculations in Front-end](https://app.shortcut.com/oazo-apps/story/2529/use-common-library-for-calculations-in-front-end)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- Use the library @oasisdex/multiply (code [here](https://github.com/OasisDEX/common/tree/main/packages/multiply)) for calculating getMultiplyParams , getCloseToDaiParams and getCloseToCollateralParams
  
## How to test

- Open multiply vault and check that the values are correct.
- Close vault with close to Dai and check the values.
- Close vault with close to collateral with conditions met to skip the flash loan (there's enough collateral to not need it), check that the flash loan is not included in transaction.
- Close vault with close to collateral with flash loan.
- Check the code.
